go.mod

```diff
@@ -33,10 +33,14 @@ require (
 	github.com/99designs/keyring v1.1.6 // indirect
 	github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d // indirect
 	github.com/DataDog/zstd v1.4.8 // indirect
+	github.com/Masterminds/goutils v1.1.1 // indirect
```

Comment:
run `go mod tidy`

Reply:
nice! By the way, how can you detect that this is still necessary before running the command?

Reply:
needs to be run after you add the proto changes with `make proto-gen`